client/components/largeCarousel.jsx

```diff
@@ -1,9 +1,21 @@
 import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
 
 function LargeCarousel(props) {
   const { picture } = props;
+  const Large = styled.div`
+    display:flex;
+    width: 100%;
+    justify-content: center;
+  `;
+  const Image = styled.img`
+    max-width: 100%;
+    height: 100%;
+  `;
   return (
-    <img src={picture.large} alt="Pending Imagery" />
+    <Large>
+      <Image src={picture.large} alt="Pending Imagery" />
+    </Large>
   );
```

Comment:
You could use more descriptive tags names like LargeImageContainer or something similar to make it easier to read for others.